package/src/components/MessageInput/FileUploadPreview.tsx

```diff
@@ -59,13 +66,63 @@ const styles = StyleSheet.create({
     marginLeft: 8,
     marginRight: 8,
   },
+  unsupportedFile: {
+    flexDirection: 'row',
+    paddingLeft: 10,
+  },
+  unsupportedFileText: {
+    fontSize: 13,
```

Comment:
13 sounds odd, did you get the font size from figma?

Reply:
✅